cloudify_agent/tests/installer/test_operations.py

```diff
@@ -152,3 +153,36 @@ def test_3_2_backwards(self, _):
 
         env.execute('uninstall', task_retries=1)
         self.wait_for_daemon_dead(name=agent_name)
+
+    @only_os('posix')
+    @only_ci
+    @patch('cloudify.workflows.local._validate_node')
+    def test_local_agent_from_source_long_name(self, _):
+        """Agent still works with a filepath longer than 128 bytes
+
+        This test won't pass on windows because some files within the
+        virtualenv exceed 256 bytes, and windows doesn't support paths
+        that long.
+        """
+        agent_name = 'agent-' + ''.join(uuid.uuid4().hex for i in range(4))
+        agent_queue = '{0}-queue'.format(agent_name)
+
+        inputs = {
+            'source_url': self.source_url,
+            'requirements_file': self.requirements_file,
+            'name': agent_name,
+            'queue': agent_queue
+        }
+
+        blueprint_path = resources.get_resource(
+            'blueprints/3_2-agent-from-source/3_2-agent-from-source.yaml')
```

Comment:
not sure we want the 3_2 agent blueprint, the 3.2 blueprint is mainly for backwards compatibility testing.  `blueprints/agent-from-source/local-agent-blueprint.yaml` is probably more suitable here